src/code_reviewer/graph.py

```
@@ -1,136 +1,107 @@
 """Graphs that extract memories on a schedule."""
 
 import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)